Preinstall Scripts Fail

```
$ npm install videojs-vast-vpaid
-
> videojs-vast-vpaid@0.1.24 preinstall /usr/local/lib/node_modules/videojs-vast-vpaid
> npm run prepare-hooks -s && npm run setup-hooks -s

Setting pre-commit hook
ln: .git/hooks/pre-commit: No such file or directory
Setting pre-push hook
ln: .git/hooks/pre-push: No such file or directory
npm ERR! Darwin 15.2.0
npm ERR! argv "node" "/usr/local/bin/npm" "link" "videojs-vast-vpaid"
npm ERR! node v0.12.7
npm ERR! npm  v2.12.1
npm ERR! code ELIFECYCLE

npm ERR! videojs-vast-vpaid@0.1.24 preinstall: `npm run prepare-hooks -s && npm run setup-hooks -s`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the videojs-vast-vpaid@0.1.24 preinstall script 'npm run prepare-hooks -s && npm run setup-hooks -s'.
npm ERR! This is most likely a problem with the videojs-vast-vpaid package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run prepare-hooks -s && npm run setup-hooks -s
npm ERR! You can get their info via:
npm ERR!     npm owner ls videojs-vast-vpaid
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lpender/dev/freelance/sv-player/npm-debug.log
```

--- a/scripts/setup-hooks
+++ b/scripts/setup-hooks
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 #DON'T SETUP THE HOOKS IF YOU ARE IN CI
-if [ "$CONTINUOUS_INTEGRATION" != "true" ]; then
+if [ "$CONTINUOUS_INTEGRATION" != "true" ] && [ -d ".git" ]; then
   echo "Setting pre-commit hook"
   ln -f -s "$PWD/scripts/pre-commit" .git/hooks/pre-commit
 

